.kokoro/build.sh

```diff
@@ -21,8 +20,20 @@ cd github/docuploader
 # Disable buffering, so that the logs stream through.
 export PYTHONUNBUFFERED=1
 
-# Run tests
-nox -s lint test
+# Debug: show build environment
+env | grep KOKORO
+
+# Setup service account credentials.
+export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/service-account.json
+
+# Setup project id.
+export PROJECT_ID=$(cat "${KOKORO_GFILE_DIR}/project-id.json")
+
+# Remove old nox
+python3.6 -m pip uninstall --yes --quiet nox-automation
```

Comment:
Can we remove this from the image instead? Surprising to have it installed only to uninstall.